src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator has started working.");
```

Comment:
The message in the `doWork` method should be more informative and indicate the type of machine that is working. For example: 'Excavator is working.'